includes/class-swedbank-pay-plugin.php

```diff
@@ -34,6 +35,13 @@ class Swedbank_Pay_Plugin {
 	const ADMIN_SUPPORT_PAGE_SLUG = 'swedbank-pay-menu-support';
 	const ADMIN_UPGRADE_PAGE_SLUG = 'swedbank-pay-menu-upgrade';
 
+	/**
+	 * Order Management.
+	 *
+	 * @var OrderManagement
+	 */
+	private $om;
```

Comment:
Suggestion: It's not really important, but short property names like this does make it a bit harder to read the code since you need to always remember what `om` means in this case. The same goes for the function you made to get the instance of the order management class. I would suggest to simply call it `order_management` instead since that will be instantly clear to anyone reading the code.